product_docs/docs/edbcloud/beta/administering_cluster/01a_superset_access.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Managing Superset Access
+---
+
+You control access to Superset data sources by Superset roles and permissions. Learn more about Superset roles, users, and permissions management in [Superset Security](https://superset.apache.org/docs/security). To assign Superset permissions to the EDB Cloud user role, see [Changing Role Permissions](01_portal_access/#changing-role-permissions).
```

Comment:
Is it worthwhile to pull a little bit more context over from the Superset documentation here? I read this whole page multiple times and still don't totally understand what's going on. Is the essential bit that we're using EDB Cloud RBAC to grant permissions to users in Superset?

Reply:
@vishalsawale9 , can you answer Ben's question?

Reply:
See my latest commit for my attempt at clarifying this topic.

Reply:
@ba-edb : 

> Is the essential bit that we're using EDB Cloud RBAC to grant permissions to users in Superset?

Yes, we're managing superset roles/permissions form EDB Cloud RBAC. I think we’re doing the right thing already if I'm seeing the correct link below. Moreover, we're giving a link to superset security page, which does talks more in details about roles and security.

https://61435e88a54b17221d5751de--edb-docs-staging.netlify.app/docs/edbcloud/latest/administering_cluster/01a_superset_access/

@drothery-edb:  We probably want revert the order of role/privileges matrix. i.e. Admin should be at top and Gamma should be the last entry of that table.

Reply:
I think the existing order makes sense since the privileges are cumulative and we refer to the information we provide in earlier rows so I didn't change it. We can discuss if you feel strongly about it.

Reply:
The thing is superset docs has exactly reverse order i.e. they first talks about admin, then alpha and so on... 
So i just wondered if we can follow similar cadence, as people generally go from Admin to read-only hierarchy. 

But whatever we're telling can work as long as the tabular information is easy to read and understand for new users.